rplidar.go

```diff
@@ -125,40 +127,17 @@ func (rp *Rplidar) NextPointCloud(ctx context.Context) (pointcloud.PointCloud, e
 		return nil, errors.New("resource (rplidar) is closed")
 	}
 
+	if !rp.started {
+		return nil, errors.New("resource (rplidar) failed to start upon initialization")
```

Comment:
Should we say "not initialized" rather than "failed to start upon initialization"?

Reply:
I'm open to the change, however, the reason I didn't go with this initially is that this error occurs after a successful:
`rplidarDevice, err := getRplidarDevice(devicePath)`
which I treated as more of the initialization procedure, while started is set once the motor is started and the `StartScan` has been called.

Reply:
Hmm, I see. I still would suggest something like "not initialized" because I would expect to see "Failed to start upon initialization" logged by the function that tried and failed to initialize, but don't feel strongly. Feel free to resolve.